### DISCUSSION TITLE
Is there a way to use a single arbitrary selector that applies to multiple classes (like `[&_input:checked~div]:`)

### DISCUSSION THREAD

**Discussion by dalvarez** [tailwindlabs/tailwindcss#13909] [Help]
I find sometimes i need to write some pseudo classes or arbitrary ones that need to apply to multiple classes. For example, I have a div with

```
[&_input:checked~div]:ml-8 [&_input:checked~div]:inline-grid [&_input:checked~div]:border-l-[1px] [&_input:checked~div]:border-gray-200 [&_input:checked~div]:pl-20
```

Is there a way to do something like this:

```[&_input:checked~div]:(ml-8 inline-grid border-l-[1px] border-gray-200 pl-20)```

almost like the distributed property in math.

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)
